### ISSUE TITLE
FR: Put more horizontal space between mouse and popup to prevent obscuring items above/below

### ISSUE THREAD

**Issue by pjeby** [nothingislost/obsidian-hover-editor#108]
A lot of places in Obsidian have vertical lists of hoverable things: File Explorer, Backlinks, Seach, Recent Files, Kanban boards, and even just lists of links.  However, because of how close to the mouse popups are placed, one can easily get into a situation where you have a shrinking amount of space to move the mouse up or down without running into a popup.  It's even worse on the right-hand sidebar, where you often end up only being able to move one direction in a list without running into a popup.

Ideally, the popup should be positioned so that its horizontal extent is always a certain minimum distance from the mouse (on the order of a cm or half-inch or so), so that one can simply move the mouse up or down a list of links without running into any popups, or having the area available to move in decrease with every new hover target.

**Comment by nothingislost** [comment]
Yeah, trying to fix this is part of what led me to do the latest decoupling work. Messing with position() was causing detect() problems but it should be much easier to change the position logic now or switch it out entirely with something like popper.js
